README.md

```diff
@@ -198,6 +198,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Aj Mendoza](https://github.com/ajmdotdev)
   - [Ajay Anuragi](https://github.com/ajayanuragi)
   - [Ajith P. Kumar](https://github.com/JOSHI-owo/)
+  - [Akash Singh](https://github.com/Akash-S1999)
```

Comment:
Please add your name in line 202, after Akash Rajak.
Let us know if you need any help on this. :)


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix